doc/api/fs.markdown

```diff
@@ -642,6 +642,9 @@ callback, and have some fallback logic if it is null.
 
 ## fs.exists(path, callback)
 
+`fs.exists()` is **deprecated**. For supported alternatives please check out
+[`fs.stat`](fs.html#fs_fs_stat_path_callback) or [`fs.access`](fs.html#fs_fs_access_path_mode_callback).
```

Comment:
@tellnes I wanted to wrap the line at 80 columns but am unsure how do wrap lines at 80 chars in the source but keep them at 80 characters in the markdown.

Is there a way to wrap the line here while not doing so in the output?